### PR TITLE
Adding license file and copyright headers

### PR DESCRIPTION
**Pull request by lizgehret**
Added missing license file and copyright headers

### CODE REVIEW

**Comment by lizgehret** [comment]
Ran flake8 and made necessary updates.

File: ci/master/template.py [hunk 1]
```diff
@@ -1,8 +1,15 @@
-#!/usr/bin/env python
+# ----------------------------------------------------------------------------
+# Copyright (c) 2017-2021, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+# !/usr/bin/env python
```
Comment (thermokarst) [inline ci/master/template.py:9]: The [hashpling](https://en.wikipedia.org/wiki/Shebang_(Unix)) needs to be at the top of the file (this tells your shell what executable to use when running the script). `q2lint` knows how to read/lint this, so you should be able to just move this up to the first line. Also, please note that it is `#!/usr/bin/env python`, not `# !/usr/bin/env python` - that little space can make a world of a difference when running the script!

```suggestion
#!/usr/bin/env python

# ----------------------------------------------------------------------------
# Copyright (c) 2017-2021, QIIME 2 development team.
#
# Distributed under the terms of the Modified BSD License.
#
# The full license is in the file LICENSE, distributed with this software.
# ----------------------------------------------------------------------------
```

File: extra/github/template.py [hunk 2]
```diff
@@ -1,11 +1,19 @@
-#!/usr/bin/env python
+# ----------------------------------------------------------------------------
+# Copyright (c) 2017-2021, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+# !/usr/bin/env python
```
Comment (thermokarst) [inline extra/github/template.py:9]: Same comment as above, I'm marking here so that it makes it onto your TODO list!

**Review by thermokarst — CHANGES_REQUESTED** [review]
This is great! I have marked two instances of a _very minor_ issue that should be resolved here prior to merging, but overall this looks great - good work!

File: extra/github/template.py (hunk 2 above)
Comment (lizgehret) [reply extra/github/template.py]: Thanks @thermokarst! Accepted and submitted your changes.

File: ci/master/template.py (hunk 1 above)
Comment (thermokarst) [reply ci/master/template.py]: Hey @lizgehret, I just edited my suggestion above, there was an extra linebreak between the hashpling and the header line, which is not valid q2lint material.

File: extra/github/template.py (hunk 2 above)
Comment (thermokarst) [reply extra/github/template.py]: Thanks @lizgehret - please see my note above - there was a typo in my code suggestion.

**Review by lizgehret — COMMENTED** [review]
Removing line breaks after hashplings

**Review by thermokarst — APPROVED** [review]
Great, thanks so much!

**Pull request merged**